### PR TITLE
feat(monitoring): change http_latency_high threshold to configurable (DT-3225)

### PR DESCRIPTION
**Pull request by christiezhao-outreach**
## Jira ID

[DT-3225]

## Notes for your reviewers

[DT-3225]: https://outreach-io.atlassian.net/browse/DT-3225?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/monitoring/datadog.tf.tpl [hunk 1]
```diff
@@ -166,6 +166,21 @@ variable http_success_rate_evaluation_window {
...
+variable http_latency_high_low_traffic_threshold {
+  type = number
+  default {{ stencil.Arg "terraform.datadog.http.latency.thresholds.lowTraffic" | default 2 }}
+}
+
+variable http_latency_high_high_traffic_threshold {
+  type = number
+  default {{ stencil.Arg "terraform.datadog.http.latency.thresholds.highTraffic" | default 2 }}
+}
+
```
Comment (jaredallard) [inline templates/monitoring/datadog.tf.tpl:183]:
```suggestion
variable http_latency_high_count_threshold {
  type = number
  default = {{ stencil.Arg "terraform.datadog.http.latency.count.highCount" | default 1000 }}
}

variable http_latency_high_low_traffic_threshold {
  type = number
  default = {{ stencil.Arg "terraform.datadog.http.latency.thresholds.lowTraffic" | default 2 }}
}

variable http_latency_high_high_traffic_threshold {
  type = number
  default = {{ stencil.Arg "terraform.datadog.http.latency.thresholds.highTraffic" | default 2 }}
}

```

File: manifest.yaml [hunk 2]
```diff
@@ -167,6 +167,18 @@ arguments:
     schema:
       type: number
     description: Overrides the high traffic threshold in the "http_success_rate_low" terraform module in monitoring/datadog.tf.
+  terraform.datadog.http.latency.count.highCount:
```
Comment (jaredallard) [inline manifest.yaml:170]: Can you add a section to the docs for these? https://github.com/getoutreach/stencil-golang/blob/main/docs/arguments.md

**Comment by malept** [comment]
I think this is either a `feat` or a `fix` instead of a refactor, I'll let @jaredallard decide which is more applicable here.

**Comment by jaredallard** [comment]
> I think this is either a `feat` or a `fix` instead of a refactor, I'll let @jaredallard decide which is more applicable here.

I'd say `feat`, personally!

File: manifest.yaml (hunk 2 above)
Comment (christiezhao-outreach) [reply manifest.yaml:170]: Does it exist for the other variables, such as `http_success_rate_high_traffic_percentile`?

Comment (jaredallard) [reply manifest.yaml:170]: Ah, sorry, I should've noted. Just at the end is fine, it was missed in past PRs but just an entry _about_ the argument is good (like the other arguments)

**Comment by malept** [comment]
Nit: I'm not a fan of putting the Jira issue in the semantic commit scope section. I would instead put the scope as the part of the repo or part of the code you're changing, and put the Jira ticket at the end, which would make the PR summary look like this:

```
feat(monitoring): change http_latency_high threshold to configurable (DT-3225)
```

File: manifest.yaml (hunk 2 above)
Comment (christiezhao-outreach) [reply manifest.yaml:170]: Added

**Review by jaredallard — APPROVED** [review]
Thank you!

**Pull request merged**